http/server.go

```diff
@@ -112,3 +113,17 @@ func GetRequestID(r *http.Request) string {
 func GetUserID(r *http.Request) string {
 	return strx.Or(r.Header.Get("X-User"), r.Header.Get("X-User-Id"))
 }
+
+// ReadSkipAndCount returns the skip and count pagination values from the query
```

Comment:
```suggestion
// GetSkipAndCount returns the skip and count pagination values from the query
```

Reply:
Can we add a sentence about what "skip" and "count" mean to the UI or a calling client?

Reply:
Done